e2e/pages/events/createOrderEvent.page.js

```diff
@@ -7,6 +7,11 @@ module.exports = {
     title: '#order_title',
     details: '#order_details',
     orderTypeList: '#orderTypeAndDocument_type',
+    directionsNeeded: {
+      yes: '#orderFurtherDirections_directionsNeeded-Yes',
+      no: '#orderFurtherDirections_directionsNeeded-No',
```

Comment:
Why do we have directionsNeeded.no if we don’t use it here?

Reply:
I like the idea of the page objects being a bit like an API for the page. i.e this is everything you can do... But sure it's not used. You think I should remove?

Reply:
hmm, maybe that is a good idea then actually. i think leave in :)